sync_ci/pkg/server/server.go

```diff
@@ -31,11 +31,12 @@ func NewServer(cfg *model.Config) *Server {
 }
 
 func (s *Server) Run() {
-	if err:=model.InitLog(s.cfg.LogPath);err!=nil{
+	if err := model.InitLog(s.cfg.LogPath); err != nil {
 		log.S().Fatalf("init log error , [error]", err)
 	}
 	httpServer := s.setupHttpServer()
 	go httpServer.ListenAndServe()
```

Comment:
pls make sure `UpdateRules` success when init app, then new a goroutine to `UpdateRulesPeriodic`